app/jobs/content/import_item_detail_job.rb

```diff
@@ -0,0 +1,9 @@
+module Content
```

Comment:
We don't need to scope any more, as we do not have the Audit Tool has been moved out of the CPM